### PR TITLE
www-apps/kibana-bin: allow using newer node.js version

### PR DESCRIPTION
**Pull request by hydrapolic**
Currently kibana 6.4.0 fails to work with node.js 8.12.0 as it takes the version string from package.json.

https://github.com/gentoo/gentoo/pull/9884

### CODE REVIEW

File: www-apps/kibana-bin/kibana-bin-6.4.0-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	newins "${FILESDIR}"/${MY_PN}.logrotate ${MY_PN}
+
+	newconfd "${FILESDIR}"/${MY_PN}.confd ${MY_PN}
+	newinitd "${FILESDIR}"/${MY_PN}.initd ${MY_PN}
+	systemd_dounit "${FILESDIR}"/${MY_PN}.service
+
+	insinto /opt/${MY_PN}
+	doins -r .
+
+	chmod +x "${ED%/}"/opt/${MY_PN}/bin/* || die
```
Comment (Zlogene) [inline www-apps/kibana-bin/kibana-bin-6.4.0-r1.ebuild]: why not `fperms` though?

Comment (hydrapolic) [reply www-apps/kibana-bin/kibana-bin-6.4.0-r1.ebuild]: I cannot remember exactly, although I tried fperms before, it seems to work, so maybe I used it the wrong way.

**Comment by Zlogene** [comment]
Oh, seems besides the singoff tag your extended commitmsg misses proper Closes and link to it (it points to the PR though, as I can see).

**Comment by hydrapolic** [comment]
@Zlogene, pushed an update, good like this?

**Comment by mgorny** [comment]
Also, please use full URL in `Closes` tag.

**Comment by mgorny** [comment]
Also, for the future:

```
 * QA Notice: Files built without respecting CFLAGS have been detected
 *  Please include the following list of files in your report:
 * /opt/kibana/node_modules/clipboardy/fallbacks/linux/xsel

**Comment by hydrapolic** [comment]
Thanks @mgorny, I'll look into that QA notice too.

**Pull request closed without merging**